components/search/AutoSuggest.vue

```diff
@@ -27,6 +27,7 @@
       :data-index="index"
       @mouseover="focus = index"
       @focus="index === focus"
+      @click="$root.$emit('emitFilterReset')"
```

Comment:
This messes up the state if you don't leave a collection page via the 'x' on the search pill.
Reproduce: 
* Go to a collection page
* Click on a search result
* Start typing something into the search box in order to get suggestions
* Click a suggestion
* Observe you'll be redirected to the search page.

Just get rid of this line and the reset filters call in the SearchInterface mounted section to fix this.

Reply:
thanks, I have switched to your solution